Cython/Utility/ModuleSetupCode.c

```diff
@@ -251,6 +251,16 @@
   #endif
 #endif
 
+#ifdef HPY
```

Comment:
I think there should probably be a separate `CYTHON_COMPILING_IN_HPY` section for this. Or would you consider it orthogonal to a specific runtime environment?

Reply:
For now, yes, I think we should have a separate `CYTHON_COMPILING_IN_HPY` section.
I'm just not sure if we will need it in a long term.
The reason for this block is that HPy does currently not support multi-phase init (and disabling it saved me a lot of work) and the Refnanny is probably not required for HPy at all since HPy comes with a debug mode. Or do you think that the Refnanny would also make sense for HPy?

Reply:
I don't think we need the refnanny in HPy, no.